mgmtfn/k8splugin/contivk8s/vagrant/export/nc-busybox/Dockerfile.nc-busybox

```diff
@@ -0,0 +1,5 @@
+FROM progrium/busybox
+RUN opkg-install netcat
```

Comment:
+1
I was unaware of opkg-install, can we use it to install other utils too? iperf3, etc.?